gym/core.py

```diff
@@ -338,8 +356,13 @@ def reset(self, **kwargs):
         return self.env.reset(**kwargs)
 
     def step(self, action):
-        observation, reward, done, info = self.env.step(action)
-        return observation, self.reward(reward), done, info
+        step_returns = self.env.step(action)
+        if len(step_returns) == 5:
+            observation, reward, terminated, truncated, info = step_returns
+            return observation, self.reward(reward), terminated, truncated, info
+        else:
+            observation, reward, done, info = step_returns
+            return observation, self.reward(reward), done, info
```

Comment:
```suggestion
        observation, reward, *rests = self.env.step(action)
        return (observation, self.reward(reward), *rests)
```

Reply:
Same comment, even though it adds a few lines, I wanted to keep this explicit.